common/src/test/java/com/revenuecat/purchases/common/BackendTest.kt

```diff
@@ -277,6 +278,92 @@ class BackendTest {
         }
     }
 
+    @Test
+    fun `postReceipt passes pricing phases as maps in body`() {
+        val purchaseOption = storeProduct.purchaseOptions[0]
+        val receiptInfo = ReceiptInfo(
+            productIDs = productIDs,
+            storeProduct = storeProduct,
+            purchaseOptionId = purchaseOption.id
+        )
+
+        mockPostReceiptResponseAndPost(
+            backend,
+            isRestore = false,
+            observerMode = false,
+            receiptInfo = receiptInfo,
+            storeAppUserID = null
+        )
+
+        val expectedPricingPhases = receiptInfo.pricingPhases
+        val expectedPricingPhasesMap = expectedPricingPhases?.map { it.toMap() }
```

Comment:
This is not really a map but a list. I would rename it to something like `mappedExpectedPricingPhases`? Idk that might not be a good name either haha, but it's a list of pricing phases represented as maps

Reply:
good point